data/sd/retired/Greg-Jamison-fe9a44c7-7241-4395-98e3-8445b1bc2d54.yml

```diff
@@ -1,21 +0,0 @@
-id: ocd-person/fe9a44c7-7241-4395-98e3-8445b1bc2d54
-name: Greg Jamison
-given_name: Greg
-family_name: Jamison
-image: https://sdlegislature.gov/inc/img/Legislators/Jamison_2017.jpg
-party:
-- name: Republican
-roles:
```

Comment:
@bfossen-ce this shouldn't be deleted. It should reflect that they served in district 12 from January 2016-December 2018. Then served again in district 16 starting on 01/12/2021 and continuing into the future.